iocore/eventsystem/I_VConnection.h

```diff
@@ -116,13 +116,16 @@
 //
 //////////////////////////////////////////////////////////////////////////////
 
-/** Used in VConnection::shutdown(). */
-enum ShutdownHowTo_t {
-  IO_SHUTDOWN_READ = 0,
-  IO_SHUTDOWN_WRITE,
-  IO_SHUTDOWN_READWRITE,
+enum class IOShutdown {
+  READ      = SHUT_RD,
+  WRITE     = SHUT_WR,
+  READWRITE = SHUT_RDWR,
 };
 
+#define IO_SHUTDOWN_READ IOShutdown::READ
```

Comment:
Good point @dragon512, thanks! I think that at least in this change it is better to keep consistency with the existing style. If we change the style we should sweep the code. However I think this might be a promising pattern for mechanically eliminating the `#defines`.